lib/runner.js

```diff
@@ -38,6 +39,7 @@ function Runner(options) {
 	this.tests = new TestCollection();
 	this._bail = options.bail;
 	this._serial = options.serial;
+	this._match = options.match;
```

Comment:
These options must be explicitly forwarded in https://github.com/sindresorhus/ava/blob/4a1bd79fdd5dd13e011750b6cb2fefc94f74b5c6/index.js#L14